File: myproject/spotifysearch.js
const fetch = require('node-fetch');

const accessToken = 'BQCnkpOJOEXQKqVyy-uR86Gyv-_QX1gQJ9PrjGt3dTHEV8Vy7FB0tL839qct58Jt7p9fcn_twYc9nb4rHwgE6IFd2NQ7Xj3GNt8SMKLfj5fkOxs7KZE'; 
const query = encodeURIComponent('Yesterday artist:The Beatles');
const url = `https://api.spotify.com/v1/search?q=${query}&type=track`;

fetch(url, {
  method: 'GET',
  headers: {
    'Authorization': `Bearer ${accessToken}`,
    'Content-Type': 'application/json'
  }
})
.then(response => response.json())
.then(data => console.log(data))
.catch(err => console.error(err));

File: myproject/spotifysearch.py
import requests

access_token = 'BQB2UWHHvPaFYJIfvjeB2Hgp-dfuXHVYn0NrTT5uPyyF24v2StqkHQyUp0yAALaIG2lwqEtCu9hbyIsIQLTMxXNQOavhzwKLt1A9x_W_z0i73XgMo4k'
query = 'Yesterday artist:The Beatles'
url = f'https://api.spotify.com/v1/search?q={requests.utils.quote(query)}&type=track'

headers = {
    'Authorization': f'Bearer {access_token}',
    'Content-Type': 'application/json'
}

response = requests.get(url, headers=headers)

try:
    data = response.json()
    print(data)
except Exception as err:
    print(err)
